TypeInjections/Tool/Tool.fs

```diff
@@ -4,39 +4,88 @@
 
 namespace Tool
 
+open LibGit2Sharp
 open System.IO
 open System
 open TypeInjections
 
 module Tool =
     
+    //NOTE 'location' refers to either the github url OR path to repo on local computer
+    let getCommit (SHA: string) (location:string) (outputPath:string) : unit =
```

Comment:
for `getCommit` I'm a bit surprised this has return type `unit`

Reply:
Yeah the name is misleading, I renamed method to `checkoutCommit`